tests/e2e/pageObjects/DicomViewer.ts

```diff
@@ -0,0 +1,65 @@
+import { Page, expect } from '@playwright/test'
+import { state } from '../hooks'
+import { config } from '../config.js'
+import util from "util";
+
+export class DicomViewer {
+    page: Page = state.page
+    elements: Readonly<Record<string, string>> = {
+        userNameSelector : '#oc-login-username',
+        passwordSelector : '#oc-login-password',
+        loginButtonSelector : 'button[type="submit"]',
+        webContentSelector : '#web-content',
+        userMenuButtonSelector : '#_userMenuButton',
+        logoutSelector : '#oc-topbar-account-logout',
+        loginFormSelector : '.oc-login-form',
+        resourceUploadButton : '#upload-menu-btn',
+        fileUploadInput : '#files-file-upload-input',
+        resourceNameSelector : '#files-space-table [data-test-resource-name="%s"]',
+        appbarResourceNameSelector : '#app-top-bar-resource [data-test-resource-name="%s"]',
+        uploadInfoCloseButton : '#close-upload-info-btn',
+        dicomCanvasSelector : '#dicom-canvas',
+        dicomVIPMetadataSelector : '#dicom-viewer-vip-metadata',
+        dicomExtendedMetadataBtnSelector : '.preview-controls .preview-controls-show-metadata',
+        dicomMetadataSidebarSelector : '#dicom-metadata-sidebar',
```

Comment:
i think you need to check if` #dicom-metadata-sidebar-content` is visible because this only becomes visible if some metadata is avilable whereas the header of the sidebar is visible even if no metadata was extracted